### ISSUE TITLE
CPD warning with version 0.9.8

### ISSUE THREAD

**Issue by phcouton** [SonarOpenCommunity/sonar-cxx#1307]
I have updated my SonarQube 6.2 with the version 0.9.8 of the cxx plugin. The plugin produces now a lot of traces during the analysis:
```
INFO: Sensor c++ SquidSensor
INFO: SonarQube 6.2 or newer environment
WARN: CPD error in file '/home/phil/.../Provider.cc' at line:395, column:47
...
```
Those warnings where not produced with the version 0.9.7 of the plugin.

It seems that this warning is produced only when C++ string constants are defined on multiple lines. Examples of two cases which produces this warning:
```c++
sql << "SELECT epsg_datum.datum_name, \
               epsg_datum.datum_code  \
                    FROM epsg_datum";
throw MyException("hello "
                  "world");
```

**Comment by guwirth** [comment]
@phcouton this was also in old versions an issue. Problem is that preprocessor is merging/changing the code and resulting line numbers does no more fit to original source file. I see no way to solve this.

**Comment by guwirth** [comment]
Remove the warning in https://github.com/SonarOpenCommunity/sonar-cxx/blob/master/cxx-sensors/src/main/java/org/sonar/cxx/sensors/visitors/CxxCpdVisitor.java#L109
Show only debug info.
